### ISSUE TITLE
Find a better place for the ads

### ISSUE THREAD

**Issue by rclai** [dmfrancisco/JS.coach#7]
:smile:

**Comment by dmfrancisco** [comment]
Yep, I'm not a fan of the current location either, but the sidebar doesn't work very well because we have to respect a given minimum width for the Ad (and the sidebar width changes according to screen size). Also, we are required to have the Ad above the fold.

The CarbonAds were added to minimize the monthly costs. Since the project is now open source, I'll also ping Digital Ocean to check if they are interested in sponsoring the project. They were very generous in the past and gave us free credits for React.parts!
